app/dashboard/utils.py

```diff
@@ -72,9 +79,15 @@ def getIPFS():
         raise IPFSCantConnectException("IPFS is not running.  try running it with `ipfs daemon` before this script")
 
 
+@backoff.on_exception(
+    backoff.expo,
```

Comment:
maybe should have a @transaction.atomic around it too so that theres no non-atomic side effects of the inner loop?

Reply:
We're not hitting the DB for `ipfs_cat` or `get_bounty`.  Where would we want to ensure atomicity here?

Reply:
my mistake.. i thought we were gonna have one large `@backoff.on_exception` wrapped around the entire sync_geth method .. but it appears that is not the case..